types/hapi__mimos/index.d.ts

```diff
@@ -2,10 +2,11 @@
 // Project: https://github.com/hapijs/mimos
 // Definitions by: AJP <https://github.com/AJamesPhillips>
 //                 Silas Rech <https://github.com/lenovouser>
+//                 Linus Unnebäck <https://github.com/LinusU>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.2
 
-import { DataStructure as MimeDbDataStructure } from 'mime-db';
+import database = require('mime-db');
```

Comment:
This line shouldn't need to be changed, `import` still works with declaration merged exports.